packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -556,19 +559,33 @@ export class ProjectService extends BaseService {
         }
     }
 
-    private async getUserAttributes(
-        user: SessionUser,
-        organizationUuid: string,
-    ) {
-        const userAttributes =
-            await this.userAttributesModel.getAttributeValuesForOrgMember({
-                organizationUuid,
+    async getUserAttributes({
```

Comment:
This method will be used to handle getting user attributes either from the req.account or the req.user.